### PR TITLE
feat: template example

### PR DESCRIPTION
**Pull request by darwintree**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING#create-a-pull-request).
- [x] **This is NOT a PR to submit translation**

- - -
This change is [Reviewable](https://reviewable.io/reviews/Conflux-Chain/conflux-documentation/410)

### CODE REVIEW

File: docs/espace/DeveloperQuickstart.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ A complete workflow for using foundry deploy contract is shown [here](./tutorial
 
 ### Remix Web IDE
 
-After compiling your contracts, the easiest way to deploy using Remix is by [setting up Metamask](./UserGuide.md), then selecting the **Conflux eSpace Testnet** network.
```
Comment (Pana) [inline docs/espace/DeveloperQuickstart.md:88]: Better make the file name unchange, so we can got a same doc URL

File: docs/espace/espace-user-guide.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  space="eSpace"
+  currency="CFX"
+  mainnetChainId="1030"
+  testnetChainId="71"
+  mainnetRpcUrl="https://evm.confluxrpc.com"
+  testnetRpcUrl="https://evmtestnet.confluxrpc.com"
+  mainnetScanUrl="https://evm.confluxscan.io"
+  testnetScanUrl="https://evmtestnet.confluxscan.io"
+  components={{
+    Faucet() {{
```
Comment (Pana) [inline docs/espace/espace-user-guide.mdx]: redundant bracket

**Review by Pana — COMMENTED** [review]
This feature is cool, if you can add a guide about how to add dual space doc, which is more friendly to new contributors

File: docs/espace/DeveloperQuickstart.md (hunk 1 above)
Comment (darwintree) [reply docs/espace/DeveloperQuickstart.md]: restored

**Review by Pana — APPROVED** [review]

**Pull request merged**